dotCMS/src/main/java/com/dotcms/auth/providers/jwt/beans/UserToken.java

```diff
@@ -65,8 +66,8 @@ public UserToken(final String id, final String subject, final String issuer, fin
      * @param subject - The subject of the token
      * @param ttlMillis - The expiration date of the token.
      */
-    public UserToken(String id, String subject, Date modificationDate, long ttlMillis) {
-        this(id, subject, ClusterFactory.getClusterId(), modificationDate,ttlMillis, ImmutableMap.of());
+    public UserToken(String id, String subject, Date modificationDate, long ttlMillis, final String skinId) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid variables with short names like id](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5164094)

Reply:
yeah, as I say in another comment, we don't need to add the `skinId` or another field to our `UserToken`.  We just need to make sure that when creating a `UserToken`, we use the value of `user.skinId` as the ID of the user token and then when validating the token, we just check to make sure that the`UserToken.id` matches the value in the `user.skinId`.